src/injectScript.ts

```diff
@@ -1,13 +1,17 @@
+import { Evolv, Experiments } from './types';
+
+
+let evolv: Evolv = (window as any).evolv;
```

Comment:
When `dom` has been added:
```suggestion
let evolv: Evolv = (window as Window).evolv;
```